README.md

```diff
@@ -1,18 +1,22 @@
-# Gettext Translation Support for `mdbook`
+# Internationalization and Rendering extensions for [mdbook](https://github.com/rust-lang/mdBook/)
```

Comment:
Please avoid links in the headings, it looks off when the color changes in the middle.